### PR TITLE
chore: remove bulk management special allow-list config

### PR DESCRIPTION
**Pull request by leangseu-edx**
**TL;DR -** removing special usage after removing it from internal configuration

JIRA: [AU-40](https://openedx.atlassian.net/browse/AU-40)
Link [PR](https://github.com/edx/edx-internal/pull/5291)

**What changed?**

- removing debris

**Developer Checklist**
- [ ] Test suites passing
- [ ] Documentation and test plan updated, if applicable
- [ ] Received code-owner approving review
- [ ] Bumped version number [package.json](../package.json)
- [ ] Approve link [PR](https://github.com/edx/edx-internal/pull/5291)

**Testing Instructions**

[ How should a reviewer test this PR? ]

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review
- [ ] I've tested the new functionality

FYI: @edx/masters-devs-gta

### CODE REVIEW

File: src/data/selectors/index.js [hunk 1]
```diff
@@ -227,8 +226,7 @@ export const shouldShowSpinner = (state) => (
  * @return {bool} - should show bulk management controls?
  */
 export const showBulkManagement = (state) => (
-  special.hasSpecialBulkManagementAccess(app.courseId(state))
-  || (tracks.stateHasMastersTrack(state) && state.config.bulkManagementAvailable)
+  tracks.stateHasMastersTrack(state) && state.config.bulkManagementAvailable
```
Comment (jansenk) [inline src/data/selectors/index.js:229]: I thought that we were moving the "track" check to the LMS and we were only checking state.config.bulkManagementAvailable? Was this changed?

Comment (nsprenkle) [reply src/data/selectors/index.js:229]: Yeah,  this is duplicate work, already available in a different PR.

**Review by nsprenkle — CHANGES_REQUESTED** [review]
This should only remove the access config. The selectors work duplicates work in 
 https://github.com/edx/frontend-app-gradebook/pull/200 which must merge before this.

**Comment by leangseu-edx** [comment]
> This should only remove the access config. The selectors work duplicates work in
> #200 which must merge before this.

Got it. I am going to close this one.

**Comment by nsprenkle** [comment]
This PR should still exist, but it should only be the removal of the ENV variables.

**Comment by leangseu-edx** [comment]
> This PR should still exist, but it should only be the removal of the ENV variables.

Got it. I just made the change

**Review by nsprenkle — APPROVED** [review]
Beautiful.

**Review by jansenk — APPROVED** [review]
